boards/silabs/starter_kits/slstk3402a/board.yml

```diff
@@ -1,5 +1,5 @@
 board:
-  name: efm32pg_stk3402a
+  name: slstk3402a
   vendor: silabs
   socs:
   - name: efm32pg12b500f1024gl125
```

Comment:
The variants shouldn't be declared in this file?

Reply:
The variants are implicitly here through the `socs` list.

Reply:
If I understand right the HWMv2 specification, the declaration above means `slstk3402a` provides one `efm32pg12` and one `efm32jg12` on the same board. @jhedberg is it also your understanding?

Reply:
> If I understand right the HWMv2 specification, the declaration above means `slstk3402a` provides one `efm32pg12` and one `efm32jg12` on the same board. @jhedberg is it also your understanding?

That's my impression too (based on just quickly looking at https://docs.zephyrproject.org/latest/hardware/porting/board_porting.html#write-your-board-yaml )

Reply:
Do @gmarull or @nordicjm have any guidance on this?

Reply:
I don't follow, this uses the `efm32pg12b500f1024gl125` soc which is soc series `efm32pg12b`, there is no other target

Reply:
@nordicjm @jerome-pouiller only commented on the first soc line. If you check the complete diff there's a second soc listed too.

Reply:
Not from what I see:
```
board:
  name: efm32pg_stk3402a
  name: slstk3402a
  vendor: silabs
  socs:
  - name: efm32pg12b500f1024gl125
```

Reply:
Might need to expand the diff to see it, but it's there :) It was actually added by you in your original HWMv2 PR for EFM32: https://github.com/zephyrproject-rtos/zephyr/pull/69231/files#diff-0cc69210ebe000a76b9ec97d154147060a804e7045d03d122b98c0a6e97a5342

Reply:
After that there's this line:
```
  - name: efm32jg12b500f1024gl125
```

Reply:
Yes that is fine, there's multiple socs supported on the board (they might all be there on a single dev board or there might only be one of a list) so you have to pick the one to build for

Reply:
Weird, Github does not allow to expand the file. Looks like a bug in the Github side.